libs/alert/src/Modal.tsx

```diff
@@ -251,7 +251,9 @@ function ModalFooter({ children, ...props }: ModalFooterProps) {
 
 function ModalIcon({ className, icon, ...props }: ModalIconProps) {
   const tokens = useTokens("Modal", props.tokens);
-  const containerClassName = cx(tokens.Icon.base, tokens.Icon.backgroundColor, className);
+  const containerClassName = cx(tokens.Icon.base, className, {
+    [tokens.Icon.backgroundColor]: !className?.includes("bg"),
```

Comment:
we could do this on a lot of places, because merging does not work 100% of the time. Maybe we could use https://github.com/dcastil/tailwind-merge

What specific use case were you targeting with this specific check?

Reply:
The token `Icon.backgroundColor` sets a background color which is then unconfigurable through `className`, that's why 
I'm checking if `className` includes 'bg'.
The options could be to either remove the token and set the default class name to `bg-green-500` or require one-off customization by default if a different background color of `Modal.Icon` is desired.